code/modules/mob/mob_vars.dm

```diff
@@ -4,6 +4,7 @@
 	animate_movement = 2
 	pressure_resistance = 8
 	throwforce = 10
+	invisibility = SEE_INVISIBLE_LIVING //Invisibility 25
```

Comment:
This will make AI unable to see any mobs

Reply:
No, AI will still see all mobs.

Reply:
Update your branch, another PR merged later will cause the behavior shawn described